src/board.py

```diff
@@ -25,11 +25,9 @@ def is_full(self, total_marks_on_board, board):
         else:
             return False
 
-    # tests are passing but not working as expected
     def is_spot_taken(self, board, user_input):
-        if (board[user_input] == self.player_one) or (
-            board[user_input] == self.player_two
-        ):
+        position_choice = board[user_input - 1]
```

Comment:
this looks like a potential private function to map a user input to an array index.  Fine for now since you only use it once. Just food for thought.

Reply:
good call - I'll keep it in mind.